### ISSUE TITLE
Bugs on fabform

### ISSUE THREAD

**Issue by andatoshiki** [derrygeoff/ough-hugo#1]
Hi, I'm the contributor to the hugo loveit theme as we chatted in the repo's issue tab, fixit theme owner is fairly welcomed for fabform to come into the theme as one easy contacting feature. But during our testing we found out that users are allowed to submit blank forms easily without any restrictions/protection methods as the image follows, the data tab will show some random blank infos which will possibly clog up your backend system maybe? If this is an original trait of your service, forget what about I say here. (Just a bit off topic would you mind creating an organization on GtiHub for fabforms in order for collaborators to join and build up projects? completely free)
![image](https://user-images.githubusercontent.com/101481353/169330925-65e48109-de38-4685-8764-e9e895b86337.png)

**Comment by derrygeoff** [comment]
Yes you are right. I will fix this so they can't submit empty forms.
Good suggestion. I will create an organization. Maybe you could create some hugo short codes so other hugo theme creators can pull the short codes into there theme
are you good at getting other contributors to come help us?

**Comment by andatoshiki** [comment]
> Yes you are right. I will fix this so they can't submit empty forms.

- Also prevent user from sending bulk forms? for example the person will have to wait for 10-15 min to send a second form after a first form was sent.
- A web-fronted bug with your official homepage, when the UA is changed to phone, the hamburger tab is not working properly on click, a dropdown menu should shown I assume. (video attached below)
- Also another suggestion, login verification emails are automatically classified as junk emails in outlook mailboxes, maybe a toast message to inform the user on checking the junk folder if they couldn't find the email? (Took me quite a while to find the login verification mail)

**Comment by derrygeoff** [comment]
Those are all great ideas. Love it. 
How did you make the .mp4 video. Is there a way I could add audio too it so I can post it in youtube

**Comment by andatoshiki** [comment]
> How did you make the .mp4 video. Is there a way I could add audio too it so I can post it in youtube

I screen recorded the video via an open sourced tool named [@screentogif](https://www.screentogif.com/), never dug into it not sure how they process the audio here. X(

**Comment by derrygeoff** [comment]
ok,

so you are expert with open source how do we attract more developers to the project
any ideas?

**Comment by andatoshiki** [comment]
> so you are expert

Haha nah sir, I'm just a high school student who self-taught web dev participated in VueJS and hugo theme developments, maybe try open source a "community" version of your service for people to self-host there own services and test out for bugs/issues, then fix or enchant base on these things finally publish a stable release. I have seen a lot of cases like this,  [@chevereto](https://chevereto.com/) is one example. Besides this you can also provide out premium plans with extra features.

**Comment by andatoshiki** [comment]
> I think you can lead this team in China. I will leave it you to come up with good ideas.

Alr sir no worries we can handle this. :)

**Comment by derrygeoff** [comment]
This the goal. I want all the people devlopers of themes to add fabform to there template. We have to make it very very easy for them and convince them to do it.

**Comment by andatoshiki** [comment]
> This the goal. I want all the people devlopers of themes to add fabform to there template. We have to make it very very easy for them and convince them to do it.

I know pretty much a lot of theme owners HEXO, HUGO, Jekyll... I'll try my best to convince them to use your product, but not implementing this feature to other blog frameworks is not so easy tbh, HEXO requires users to publish an npm package and jekyll plugins requires ruby on rails. I'll try work out most of the popular themes I know.

**Comment by andatoshiki** [comment]
Hey sir, would you mind if you configure a `CNAME` dns record of `doc.fabform.io` secondary domain to `cname.vercel-dns.com` by any chances? I would like to write documentations via docsify documentation builder instead of a GitHub `README.md` file. Docisifyt is way more painlet, and the resut will be deployed on Vercel, completely free.

**Comment by derrygeoff** [comment]
Do what is good

**Comment by derrygeoff** [comment]
Explain why this is good I don't see

**Comment by andatoshiki** [comment]
Ok actually nvm that, but a short demo is out in the video below, I'll release it on a hugo loveit theme based site for ppl to view after a quick nap, stayed up all night.
https://user-images.githubusercontent.com/101481353/169412073-c4683838-dca7-40cb-9b7e-3d2093c246cb.mp4

**Comment by derrygeoff** [comment]
Hello

How is the progress going with the templates?
any published yet?

**Comment by derrygeoff** [comment]
I have created a new organization for to store the templates

https://github.com/fabform-io

**Comment by andatoshiki** [comment]
> I have created a new organization for to store the templates
> 
> https://github.com/fabform-io

Please try invite me to the org either via email `andatoshiki@oulool.com` or using username `andatoshiki`.

**Comment by derrygeoff** [comment]
I fixed it so you cant submit a form with no fields an also I mentioned to check your spam folder when sending the confirmation email.
